converter/converter.go

```diff
@@ -0,0 +1,437 @@
+package converter
```

Comment:
nit: we've named the others `generate` and `format`, so I'd call this `convert`.